modules/gateway/peers_test.go

```diff
@@ -12,12 +12,25 @@ import (
 	"github.com/NebulousLabs/Sia/modules"
 )
 
+// dummyConn implements the net.Conn interface, but does not carry any actual
+// data. It is passed to muxado, because passing nil results in segfaults.
+type dummyConn struct {
```

Comment:
A mock is a specific type of test object that has programmable expectations. This is more of a "stub" than a mock, at least going by the definitions in http://martinfowler.com/articles/mocksArentStubs.html. The "dummy-" prefix is just my go-to for objects that are the bare minimum.